DigitalLearningSolutions.Data.Tests/Services/RecommendedLearningServiceTests.cs

```diff
@@ -307,10 +295,10 @@ public async Task
         }
 
         [Test]
-        [TestCase(0, 0)]
-        [TestCase(1, 4)]
-        [TestCase(2.4, 9.6)]
-        [TestCase(5, 20)]
+        [TestCase(0, 175)]
+        [TestCase(1, 179)]
+        [TestCase(2.4, 184.6)]
+        [TestCase(5, 195)]
         public async Task
             GetRecommendedLearningForSelfAssessment_returns_correct_recommendation_score_for_resource_with_learning_hub_ratings_only(

```

Comment:
This test title mentions "...correct score for {factor} only" but the expected scores seem to be higher than just the individual factor's contribution, so might be worth renaming.

Reply:
This was a consequence of me not realising the relationship was 1-1-0 (The Single comment above)
We now have 2 basic situations, either we can only calc the score from the LH rating, or we can have the LH rating + attempt both other values (req adj may be 0).
Restored these back to just the LH calculation, dropping the QuestionParameters setup.

Reply:
Ah I see, the case where no QuestionParameters are set up is handled here. I think given that that case is specifically considered in the logic for determining what to hide, it might be good to specify it in the title here then. Just will make it easier to see that it has indeed be tested from a quickly glance. 
Maybe something like "...for_resource_only_with_learning_hub_rating_and_no_question_parameters"?